### ISSUE TITLE
[FEATURE] Ability do not send some attachment types to RP after test is completed

### ISSUE THREAD

**Issue by Shakevg** [reportportal/agent-net-vstest#59]
Sometimes we have big attachments generated during the test run and it requires a lot of space to store them on RP.
But we have a local copy or copy on CI tool (like Azure DevOps) of such attachment.
It will be great to have the ability to exclude some attachment extensions that send to RP in the RP configuration file.

**Comment by nvborisenko** [comment]
I don't think we should implement this possibility. It's good whole report is coming to RP.
But this issue should be resolved by external extension. So moving it to https://github.com/reportportal/commons-net repository where extension point should be exposed to provide cancelling attachments based on criteria.
